### PR TITLE
5 Solana APIs now enabled

### PR DESCRIPTION
**Pull request by conor2e**
## Description
5 heavy Solana APIs were enabled in ash1 and ams1 rate limited at 2RPS. Already implemented by dev so Nare requested doc updates asap. 

Only 5 APIs pages needed to have the Information note at the top updated.

Content reviewed and approved by PM Vladimir Feofilaktov in Slack. 

## Related Issue

https://chainstack.myjetbrains.com/youtrack/issue/DOCS-36/FVM-testnet-user-documentation

### CODE REVIEW

**Comment by conor2e** [comment]
Needs version and release note updates.

File: docs/api/solana/getmultipleaccounts.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ meta:
 
 ::: tip Information
 
-This method is currently disabled for elastic Solana devnet nodes.
+This method is now enabled for elastic Solana devnet nodes on US Ashborne (ash1) and Amsterdam (ams1) with a rate limit of 2 requests per second (RPS).
```
Comment (akegaviar) [inline docs/api/solana/getmultipleaccounts.md:13]: https://www.datacenters.com/amazon-aws-ashburn

Comment (akegaviar) [inline docs/api/solana/getmultipleaccounts.md:13]: I don't think we should use `now` as there is no date here for anyone consuming this note. eg if two months, what would this `now` mean?

**Review by akegaviar — CHANGES_REQUESTED** [review]
changes

File: docs/api/solana/getmultipleaccounts.md (hunk 1 above)
Comment (easeev) [reply docs/api/solana/getmultipleaccounts.md:13]: In this case we're not hosted in AWS but I guess @akegaviar your point is Ashborne -> Ashburn

**Review by ginstagram — APPROVED** [review]
LGTM

**Review by akegaviar — APPROVED** [review]

**Pull request merged**